tests/TestCase/Utility/XmlTest.php

```diff
@@ -247,6 +247,39 @@ public function testBuildEmptyTag()
         }
     }
 
+    /**
+     * testLoadHtml method
+     *
+     * @return void
+     */
+    public function testLoadHtml()
+    {
+        $html = CORE_TESTS . 'Fixture/sample.html';
+        $html = file_get_contents($html);
+        $xml = Xml::loadHtml($html);
+        $xml = html_entity_decode($xml->asXML(), ENT_NOQUOTES, 'UTF-8');
+        $this->assertEquals($html, $xml);
+
+        $xml = Xml::loadHtml($html, ['parseHuge' => true]);
```

Comment:
These tests are looking good. There should be a teat case covering the domdocument return tyoe as well.